04_setup_ironic.sh

```diff
@@ -147,7 +147,7 @@ if  sudo podman pod exists ironic-pod ; then
 fi
 
 # Create pod
-sudo podman pod create -n ironic-pod
+sudo podman pod create -n ironic-pod --infra-image k8s.gcr.io/pause:3.2
```

Comment:
what's the reason to not use the default pause container?

Reply:
This was left over from testing